opencv_apps/src/nodelet/simple_flow_nodelet.cpp

```diff
@@ -113,7 +113,15 @@ class SimpleFlowNodelet : public nodelet::Nodelet
     try
     {
       // Convert the image into something opencv can handle.
-      cv::Mat frame = cv_bridge::toCvShare(msg, msg->encoding)->image;
+      cv::Mat frame_src = cv_bridge::toCvShare(msg, msg->encoding)->image;
```

Comment:
because this line convert ros msg to opencv Mat, so we should set
msg->encoding.
int this node, we use frame_src as opencv Mat with input encocding and
frame for bgr8 image for output

--
◉ Kei Okada


On Sun, Jan 3, 2016 at 4:54 AM, Vincent Rabaud <notifications@github.com>
wrote:

> In opencv_apps/src/nodelet/simple_flow_nodelet.cpp
> <https://github.com/ros-perception/vision_opencv/pull/100#discussion_r48686970>
> :
>
> > @@ -113,7 +113,15 @@ class SimpleFlowNodelet : public nodelet::Nodelet
> >      try
> >      {
> >        // Convert the image into something opencv can handle.
> > -      cv::Mat frame = cv_bridge::toCvShare(msg, msg->encoding)->image;
> > +      cv::Mat frame_src = cv_bridge::toCvShare(msg, msg->encoding)->image;
>
> why not use an encoding of "bgr8" here ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ros-perception/vision_opencv/pull/100/files#r48686970>
> .
>